packages/storage/test/blob.test.ts

```diff
@@ -42,16 +42,18 @@ describe('Firebase Storage > Blob', () => {
   it('Slicing works', () => {
     const blob = new FbsBlob(new Uint8Array([1, 2, 3, 4, 5, 6, 7]));
     const sliced = blob.slice(1, 5);
+    assert.isNotNull(sliced);
```

Comment:
Since this is a test, I think it's safe to just define `sliced` like this:

```ts
const sliced = blob.slice(1, 5)!;
```

Then you can remove the `!`s below.

Reply:
Done.